gems/aws-sdk-translate/spec/translate_document_encoding_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative 'spec_helper'
+require_relative '../lib/aws-sdk-translate/plugins/translate_document_encoding'
```

Comment:
Is this still necessary to require? Other plugins don't need this so maybe an issue.

Reply:
Moved this to an autoload in the customizations.